sc/social/like/tests/test_subscribers.py

```diff
@@ -220,6 +222,57 @@ def test_validate_with_package_uninstalled(self):
         # should not raise exceptions on editing
         notify(EditFinishedEvent(self.news_item))
 
+    @requests_mock.mock()
+    def test_validate_facebook_prefetch_valid(self, m):
+        RESPONSE_VALID = """{"share":{"comment_count":0,"share_count":4},
```

Comment:
this is not the kind of response I see from Facebook:

```python
(Pdb) url = 'https://graph.facebook.com/?id=http://plone.org/&scrape=true'
(Pdb) resp = requests.post(url)
(Pdb) pp resp.json()
{u'description': u'',
 u'id': u'10150450110122918',
 u'image': [{u'type': u'image/png', u'url': u'https://plone.org/logo.png'}],
 u'site_name': u'Plone: Enterprise Level CMS - Free and OpenSource - Community Driven - Secure',
 u'title': u'Plone CMS: Open Source Content Management',
 u'type': u'website',
 u'updated_time': u'2017-09-27T22:56:54+0000',
 u'url': u'https://plone.org/'}
```

Reply:
Altered